web/templates/settings/custom_user_profile_field.hbs

```diff
@@ -4,28 +4,28 @@
     <div class="field_hint">{{ field.hint }}</div>
     <div class="field">
         {{#if is_long_text_field}}
-        <textarea maxlength="500" class="custom_user_field_value settings_textarea">{{ field_value.value }}</textarea>
+        <textarea maxlength="500" class="custom_user_field_value settings_textarea" name="{{ field.name }}">{{ field_value.value }}</textarea>
         {{else if is_select_field}}
-        <select class="custom_user_field_value {{#if for_manage_user_modal}}modal_select{{else}}settings_select{{/if}} bootstrap-focus-style">
+        <select class="custom_user_field_value {{#if for_manage_user_modal}}modal_select{{else}}settings_select{{/if}} bootstrap-focus-style" name="{{ field.name }}">
             <option value=""></option>
             {{#each field_choices}}
                 <option value="{{ this.value }}" {{#if this.selected}}selected{{/if}}>{{ this.text }}</option>
             {{/each}}
         </select>
         {{else if is_user_field }}
-        <div class="pill-container person_picker">
+        <div class="pill-container person_picker" name="{{ field.name }}">
             <div class="input" contenteditable="true"></div>
         </div>
         {{else if is_date_field }}
-        <input class="custom_user_field_value datepicker {{#if for_manage_user_modal}}modal_text_input{{else}}settings_text_input{{/if}}" data-field-id="{{ field.id }}" type="text"
+        <input class="custom_user_field_value datepicker {{#if for_manage_user_modal}}modal_text_input{{else}}settings_text_input{{/if}}" name="{{ field.name }}" data-field-id="{{ field.id }}" type="text"
```

Comment:
I guess we can have these changes to have `name` attribute in a separate commit.

Reply:
Done.